test/nightly/azureResourceWebAppActions.test.ts

```diff
@@ -100,6 +104,15 @@ suite('Web App actions', async function (this: Mocha.Suite): Promise<void> {
         assert.equal(createdApp.scmType, constants.ScmType.None, `Web App scmType's property value should be ${constants.ScmType.None} rather than ${createdApp.scmType}.`);
     });
 
+    test('Add new setting', async () => {
```

Comment:
Could you include which OS is being used to run this action test?  Along with the other tests above?  It'll be helpful if the test fails to know which OS it failed on 😄